config/application.rb

```diff
@@ -40,9 +40,5 @@ class Application < Rails::Application
     end
     # use rack-attack
     config.middleware.use Rack::Attack
-
-    # Disable cookies, we don't use them here
-    config.middleware.delete ActionDispatch::Cookies
-    config.middleware.delete ActionDispatch::Session::CookieStore
```

Comment:
Session cookies are encrypted. Can we use a normal cookie so that it is not encrypted and users can see what is in it?

Reply:
As far as I understand, cookies do not handle serialization so the array of ids would have to be JSON encoded or similar. I thought you wanted to use cookies for convenience, not for transparancy, but if you want to switch to cookies that's OK.

Reply:
I think it would be good to switch to unencrypted cookies for transparency. Ideally we wouldn't have any cookies at all.

Reply:
I think this session cookie line can be removed again?